src/Config.hs

```diff
@@ -684,6 +717,16 @@ optionsParser = Options
                "Pass 0 to disable the limit. " ++
                "Also configurable through VAULTENV_MAX_CONCURRENT_REQUESTS.")
 
+    duplicateVariableBehavior
+      =  option (Just <$> auto)
+      $  long "duplicate-variable-behavior"
+      <> value Nothing
+      <> metavar "error | keep | overwrite"
+      <> help ("Disable the error around duplicate variables. If a variable " ++
+               "already exists in the environment, that variable is takes instead" ++
+               " of the secret. Default: disallow duplicate variables" ++
```

Comment:
> Disable the error around duplicate variables

Not quite true since you could pass `error`. I'd just explain what each of the options do and state what the default is. No need to be roundabout in the description when you can be direct.

Reply:
I rewrote the stuff, basically forgot to update the old description :see_no_evil: